content/en/blog/news/roadmap-july-2019.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "OpenCue roadmap"
+linkTitle: "OpenCue roadmap"
+date: 2019-07-08
+description: "Learn more about the OpenCue roadmap"
+---
+
+If you're attending SIGGRAPH 2019 in Los Angeles, you can learn more about
+the roadmap plans for OpenCue at an hour long Birds of a Feather session on
+July 30.
+
+The Academy Software Foundation (ASWF) will provide boxed lunches for this
+session.
+
+For more information, and to add the session to your agenda, visit the
+[event page](https://s2019.siggraph.org/presentation/?id=bof_122&sess=sess297).
```

Comment:
This link is updated now with the correct time, as is the ASWF schedule that you have linked below.

Reply:
Thanks and updated with a mention of the start time.